ClientTests/TabManagerTests.swift

```diff
@@ -459,4 +459,136 @@ class TabManagerTests: XCTestCase {
         delegate.verify("Not all delegate methods were called")
     }
     
+    func testRemoveOnlyTab() {
+        let profile = TabManagerMockProfile()
```

Comment:
Could we move 
```
let profile = TabManagerMockProfile()
        let manager = TabManager(prefs: profile.prefs, imageStore: nil)
        let delegate = MockTabManagerDelegate()
```

to `setUp()`?

Looks like it's used the same in every method

Reply:
Done! :)